README.md

```diff
@@ -5,28 +5,35 @@ Open Artemis IDE plugin for the programming exercise integration.
 This plugin integrates the [Artemis](https://github.com/ls1intum/Artemis) interactive learning platform into the IntelliJ IDE.
 It allows you to directly import programming exercises from ArTEMiS and submit your changes to the build servers.
 
-**Current version:** 1.1.1
+**Current version:** 1.1.2
 
-## Planned features
+## Example Usage
+![](.github/media/orion_workflow.gif)
+
+## Planned Features
 We want to integrate the following features into the plugin:
 
- - Automated exercise project import
- - Automated code submission (`git pull` + `git stage` + `git commit` + `git push`)
- - Java/JS bridge communication between the plugin, and the ArTEMiS Angular web application
- - Display of test results and submission feedback
- - Integration of the  [JetBrains Edu Tools](https://plugins.jetbrains.com/plugin/10081-edutools)
+- Support for manual assessment  
+- Integration of the  [JetBrains Edu Tools](https://plugins.jetbrains.com/plugin/10081-edutools)
+- Support for team exercises
+- Automatic conflict resolution
 
 ## Run/debug the plugin
 
- - Select the JBR `>11.0.3` as the project SDK with language level 11
- - Run Gradle task: `runIde`
+- Import as gradle project
+- Select the JBR `>11.0.3` as the project SDK with language level 11
+- Run Gradle task: `runIde`
 
-## Installation of a release (.zip)
-![](gifs/orion_installation.gif)
+## Testing of pull requests
 
-## Example Usage
-![](gifs/orion_workflow.gif)
+1. **Download release (.zip)**  
+  Download the release file from the pull request's Checks &rarr; Artifacts &rarr; orion
+![](.github/media/download_release.png)
 
-## Feedback? Questions?
-Email: alexander(dot)ungar(at)tum(dot)de
+2. **Install release (.zip)**  
+  Install the release file in IntelliJ at Settings &rarr; Plugins &rarr; Settings &rarr; Install Plugin from Disk... &rarr; Select the file
+![](.github/media/install_release.png)
+   [Installation process as gif](.github/media/orion_installation.gif)
```

Comment:
This displays as a link:
![image](https://user-images.githubusercontent.com/72132281/118192576-a68e8300-b446-11eb-8905-7baa754dc3b8.png)

I think you need to add the exclamation mark in front:
```suggestion
   ![Installation process as gif](.github/media/orion_installation.gif)
```

Reply:
I see. The thing is I was trying to replace the gif with the 2 new images (which in my opinion are a bit faster to read since you don't have to keep up with the gif's speed and wait till it cycles back to your position), but didn't want to delete/remove any reference to the gif. Therefore I kept it there for reference if anyone is interested. Would it be preferrable to comment it as such, delete it alltogether or keep it there and remove the image above?

Reply:
I would just remove the gif if you images already cover what it tries to show.

Reply:
I don't have a strong opinion on this, but it might be a solution to rename the link to something more descriptive like "Additional gif of the installation process", which is collapsed when the page is opened but can be uncollapsed in case somebody needs the extra information of the gif?

Reply:
I now replaced it with a collapsible so anyone who cannot follow the images can still fall back to it